flexget/plugins/output/pushalot.py

```diff
@@ -31,12 +31,15 @@ class OutputPushalot(object):
     Configuration parameters are also supported from entries (eg. through set).
     """
     default_body = "{% if series_name is defined %}{{tvdb_series_name|d(series_name)}} " \
-                   "{{series_id}} {{tvdb_ep_name|d('')}}{% elif imdb_name is defined %}{{imdb_name}} "\
+                   "{{series_id}} {{tvdb_ep_name|d('')}}{% elif imdb_name is defined %}{{imdb_name}} " \
                    "{{imdb_year}}{% else %}{{title}}{% endif %}"
     schema = {
         'type': 'object',
         'properties': {
-            'token': one_or_more({'type': 'string'}),
+            'token': {'oneof': [
+                {'type': 'string'},
+                {'type': 'array', 'items': 'string'}
+            ]},
```

Comment:
I wasn't actually aware of that, so yeah, keeping it makes perfect sense. I upgraded the schema in the lastest commit to have the ability to use just a string without an explicit token property. Also tested it all using 1 & 2 tokens and everything is peachy.